packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -956,6 +988,10 @@ class RenderParagraph extends RenderBox
         newChildren.add(newChild);
       }
     }
+    // Makes sure we annotated all of the semantics children.
+    assert(childIndex == children.length);
```

Comment:
I think I've applied the right patch, but it is now asserting on this line for the case where it wasn't crashing before.

For the case that was crashing, childIndex == children.length == 0, which is where it used to have 7 children expecting 8.  Immediately after that case, childIndex comes out == 0 but children.length == 8 and this asserts.

Reply:
will reach out offline

Reply:
update: patch wasn't right.

Confirmed this fixes the crash.